.editorconfig

```diff
@@ -8,11 +8,12 @@ end_of_line = LF
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-; 4-column tab indentation and .NET coding conventions
-[*.cs]
+; 4-column tab indentation
+[*.{cs,csproj,yaml,lua,sh,ps1}]
```

Comment:
Actually, this probably shouldn't mention `csproj`?

Reply:
Clarified on Discord. (Using tab indentation is welcomed, although we don't enforce it yet.)